modin/pandas/test/test_io.py

```diff
@@ -1656,6 +1656,21 @@ def test_read_parquet_2462(self, engine):
 
             df_equals(test_df, read_df)
 
+    def test_read_parquet_5509(self, engine):
```

Comment:
Should we add `warns_that_defaulting_to_pandas`? https://github.com/modin-project/modin/blob/e92847c03fa0d0c5cc010f8b66a2356573568a1f/modin/test/test_utils.py#L245

Reply:
good catch, added